src/pages/GratefulPage.js

```diff
@@ -1,47 +1,60 @@
 import React, { useState, useEffect } from "react";
-import firebase from '../firebase'
+import firebase from "../firebase";
 import GratefulForm from "../components/GratefulForm";
-import PhoneNumberAuth from '../components/PhoneNumberAuth'
+import PhoneNumberAuth from "../components/PhoneNumberAuth";
+import DateHeader from "../components/DateHeader";
 import moment from "moment";
 
-const firestore = firebase.firestore()
+const firestore = firebase.firestore();
+
+const headerTitle = {
+  title: moment().format(`dddd`),
+  subtitle: moment().format(`DD MMMM`)
+};
+
+const emojis = [
+  `😌`,
+  `💃`,
+  `🙏`,
+  `🎉`,
+]
+
+// Generate random emoji
+const emoji = emojis[~~(emojis.length * Math.random())];
```

Comment:
omg what did you do, it looks amazing but never saw `~~`

Reply:
That floors it to number?

Reply:
Yeah it converts the operand to the closest integer